tests/phpunit/ParserFunctions/SubobjectParserFunctionTest.php

```diff
@@ -627,7 +627,7 @@ public function tokuFixedParameterProvider() {
 				],
 				'propertyValues' => [
 					'Foo Bar',
-					'1001'
```

Comment:
Updated expectations here and in annotator tests to account for a25c0263b376abe6858eaee627a18b8c189c3e58.